recipes/pykube-ng/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Ping! Also, try to use grayskull next time. It really helps to get the recipe in good  shape for conda-forge and reduces the work of the reviewers.

Reply:
Oh nice. Wasn't aware of that and missed the suggestion in the comments.